src/theme/deployment/components/_display-group.scss

```diff
@@ -1,13 +1,28 @@
-$background-banner-primary: var(--display-group-background-banner-primary, var(--gradient-primary-light-vertical));
-$background-banner-secondary: var(--display-group-background-banner-secondary, var(--gradient-yellow-vertical));
-$background-tool-1: var(--display-group-background-tool-1, var(--ion-color-secondary-400));
-$background-tool-2: var(--display-group-background-tool-2, var(--ion-color-secondary-500));
+$background-banner-primary: var(
+  --display-group-background-banner-primary,
+  var(--gradient-primary-light-vertical)
+);
+$background-banner-secondary: var(
+  --display-group-background-banner-secondary,
+  var(--gradient-yellow-vertical)
+);
```

Comment:
I didn't change these lines of code, not sure where these additional line breaks come from...

Reply:
These line breaks for lines over a certain length are enforced by our automatic code formatter. I would guess that these files haven't been committed under the latest iteration of the formatter, so are only just having this styling applied. In any case, it's just a convention to make long lines easier to read so all good